ingestion/src/metadata/utils/pandas_utils.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+Module centralising pandas methods
+"""
+from metadata.ingestion.source.database.datalake.metadata import ometa_to_dataframe
+
+
+def return_ometa_dataframes(
+    service_connection_config, client, table, profile_sample_config
+):
+
+    return ometa_to_dataframe(
+        config_source=service_connection_config.configSource,
+        client=client,
+        table=table,
+        profile_sample_config=profile_sample_config,
+    )
```

Comment:
I think this can be removed, we don't do anything more in this function and just call `ometa_to_dataframe` directly

Reply:
added sampling to this method, we are using it for mocking and returning list[df] for tests

Reply:
Same comment than here, https://github.com/open-metadata/OpenMetadata/pull/10603#discussion_r1158587065